.agents/skills/fern-recipe-page/scripts/check_recipe_page.py

```
@@ -0,0 +1,474 @@
+    the selected value. Mirrors the `[data-x]:not([data-x~="v"])` CSS."""
+    return all(
+        selection[dim] in block.get(dim, selection[dim]).split() for dim in selection
+    )
+
+
+def reachable(dims, needs) -> list[dict[str, str]]:
+    """Combinations a reader can actually click to, honouring data-needs-*."""
+    keys = sorted(dims)
+    out = []
+    for combo in itertools.product(*[dims[k] for k in keys]):
+        sel = dict(zip(keys, combo))
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)